srm_tools/processors.py

```diff
@@ -0,0 +1,23 @@
+import dataflows as DF
+
+
+def ensure_fields(field_map, resources=None):
```

Comment:
This actually adds all the fields, regardless of if it already exists in the resource or not.